client/blocks/login/lost-password-form.jsx

```diff
@@ -47,6 +63,33 @@ const LostPasswordForm = ( {
 	const onSubmit = async ( event ) => {
 		event.preventDefault();
 
+		const accountType = await getAuthAccountTypeRequest( email );
```

Comment:
This is being called even with feature flag disabled, I think we should encapsulate it inside a feature flag check

Reply:
@ilyasfoo Good catch 👍  Fixed it in [Check the account type only when the feature flag is enabled](https://github.com/Automattic/wp-calypso/pull/94384/commits/656689d09556ac37bb00ae88a90d38538f27e6c9)